frontend/src/components/forms/CreateEmployee.tsx

```diff
@@ -0,0 +1,320 @@
+                    <FormLabel>Last Name</FormLabel>
+                    <Input
+                      placeholder="Enter last name"
+                      value={invitedLastName}
+                      onChange={handleLastNameChange}
+                      maxLength={50}
+                    />
+                    <FormErrorMessage>Last Name is required.</FormErrorMessage>
+                  </FormControl>
+                </Box>
+              </Box>
+              <Box display="flex" flexDirection="row" mt="16px">
+                <FormControl isRequired isInvalid={invitedEmailError}>
+                  <FormLabel>SHOW Email</FormLabel>
+                  <Input
+                    placeholder="Enter SHOW email"
+                    value={invitedEmail}
+                    onChange={(event) => setInvitedEmail(event.target.value)}
+                    maxLength={254}
+                  />
+                  <FormErrorMessage>
+                    A validwh email is required.
+                  </FormErrorMessage>
+                </FormControl>
+              </Box>
+            </Box>
+            <Box
+              pt="17px"
+              pb="7px"
+              pl="24px"
+              pr="24px"
+              borderBottom="1px solid #EBECF0"
+            >
+              <FormControl isRequired isInvalid={invitedAdminStatusError}>
+                <RadioGroup
+                  value={invitedAdminStatus}
+                  onChange={handleAdminStatusChange}
+                  pb="24px"
+                >
+                  <Flex>
+                    <Box marginRight="24px">
+                      <Radio size="lg" colorScheme="teal" value="1">
+                        <Text color="#1B2A2C">Admin</Text>
+                      </Radio>
+                    </Box>
+                    <Box>
+                      <Radio size="lg" colorScheme="teal" value="2">
+                        <Text color="#1B2A2C">Non Admin</Text>
+                      </Radio>
+                    </Box>
+                  </Flex>
+                  <FormErrorMessage>A selection is required.</FormErrorMessage>
+                </RadioGroup>
+              </FormControl>
+              <Checkbox
+                size="lg"
+                colorScheme="teal"
+                borderRadius="40px"
+                isChecked={isTwoFactorAuthenticated}
+                isDisabled={isDisabled}
+                onChange={(event) =>
+                  setIsTwoFactorAuthenticated(event.target.checked)
+                }
+              >
+                <Text color="#1B2A2C">Require Two Factor Authentication</Text>
+              </Checkbox>
+              <Text fontSize="12px" color="#1B2A2C" fontWeight="bold">
+                Requiring Two Factor Authentication means the employee will only
+                be able to access the platform while physically in the main
+                building
+              </Text>
+            </Box>
+            <Box
```

Comment:
Instead of using a box you could simplify it using `<ModalFooter></ModalFooter>` but it would need to be outside of the `ModalBody` tags. This is what it would look like:

![image](https://github.com/uwblueprint/supportive-housing/assets/43042601/fef6ac2f-cdad-4694-a5cc-75b8c5dadfb9)